frontend/packages/integration-tests-cypress/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
+  - rhamilto
+  - TheRealJon
```

Comment:
I think it would be good to add @zherman0 as another reviewer here. He is interested in helping our with the integration tests!

Reply:
done